tensilelite/Tensile/KernelWriter.py

```diff
@@ -2680,12 +2680,14 @@ def initKernel(self, kernel, tensorParametersA, tensorParametersB):
     if kernel["ProblemType"]["Sparse"] and not kernel["DirectToVgprSparseMetadata"]:
       vwm = kernel["GlobalReadVectorWidthMetadata"]
 
-    if not kernel["UnrollMajorLDSA"]:
+    # use lrwvTile = 1 for XFP32
+    isXFP32 = kernel["EnableF32XdlMathOp"] and kernel["ProblemType"]["F32XdlMathOp"].isXFloat32()
+    if not kernel["UnrollMajorLDSA"] and not isXFP32:
       self.states.lrvwTileA = kernel["VectorWidthA"]
     else:
       self.states.lrvwTileA = 1
 
-    if not kernel["UnrollMajorLDSB"]:
+    if not kernel["UnrollMajorLDSB"] and not isXFP32:
```

Comment:
Please add # FIXME for easy search.

Reply:
I feel FIXME is not exactly expressing this.
This specific case (XFP32+ TLU + VW) requires extra logic, but that logic is simply not implemented yet.
If you would like to put some comment, I can put TODO here.
like this,
   TODO: need to implement extra logic (swap vgprs) to support XFP32+ TLU + VW

Is this OK for you?

Reply:
It is not due to incorrect logic to fix, but the implementation itself is missing.

Reply:
I added TODO comment.